dsgrid/config/dimension_mapping_base.py

```diff
@@ -168,6 +168,16 @@ class DimensionMappingBaseModel(DSGBaseModel):
         title="to_dimension",
         description="To dimension",
     )
+    from_fraction_tolerance: Optional[float] = Field(
```

Comment:
Let's try to not use `Optional` in any case where there is a default value. `Optional` in this case is equivalent to `Union[None, float]`. In reality, this can only be a float and we don't want to allow `None`. There could be cases where we still want to allow `None`.

Reply:
Ok so if we have a default value, we don't need optional? 10-4